libdevcore/StringUtils.h

```diff
@@ -72,4 +75,78 @@ std::string joinHumanReadable
 	return result;
 }
 
+/// Returns decimal representation for smaller numbers; hex for large numbers.
```

Comment:
Please first describe the general concept (something like "formats numbers to be easily readable by humans") and then explain the details.